src/vm/i386/asmhelpers.S

```diff
@@ -472,10 +472,12 @@ NESTED_END CallDescrWorkerInternal, _TEXT
 #ifdef _DEBUG
 // int __fastcall HelperMethodFrameRestoreState(HelperMethodFrame*, struct MachState *)
 LEAF_ENTRY HelperMethodFrameRestoreState, _TEXT
+    #define     RETN 8
```

Comment:
I also though that FASTCALL incurs no change on stack, but I found the following lines while digging segmentation fault issue:
```
   0xf6fbc91e <+238>:   call   0xf71551ac <LazyMachStateCaptureState>
   0xf6fbc923 <+243>:   sub    $0x4,%esp
   0xf6fbc926 <+246>:   mov    %eax,0x80(%esp)
```
This code seems to recover the stack after calling ``LazyMachStateCaptureState``.

In addition, the corresponding assembly in Windows has the following form:
```
; int __fastcall LazyMachStateCaptureState(struct LazyMachState *pState);
@LazyMachStateCaptureState@4 proc public
```
As I know, @LazyMachStateCaptureState@4 means that LazyMachStateCaptureState will cleans up stack by 4 byte.